docs/pages/ingesting_publications.rst

```diff
@@ -0,0 +1,26 @@
+Ingesting publications
+=======================
+Ingesting a publication is a process of creating a new publication in the database.
+The publication is created based on the metadata provided by the user.
+
+`astrodb_utils` can query the `NASA Astrophysics Data System <https://ui.adsabs.harvard.edu/>`_ with the `ingest_publications` function.
+To use this feature, you'll need to set up an ADS token and add it to your environment.
+
+Set up ADS token
+-----------------------
+
+1. Make an ADS account at `https://ui.adsabs.harvard.edu/help/api/`.
+2. Go to `https://ui.adsabs.harvard.edu/user/settings/token`.
+3. Copy the token generated on the package.
+4. Add the token to your environment by running
+
+    .. code-block:: bash
+
+        export ADS_TOKEN=<your token>
+
+replacing <your token> with the token you copied.
```

Comment:
I would perhaps rephrase that as "Add the token as an environment variable ADS_TOKEN. If using bash, this can be done with:" and then have the code block. The reason being that not everyone uses bash for the shell and they may have more permanent places where they store environment variables (like a .bashrc file).

Reply:
I don't think we need to cover all possible ways to set environment variables or how to create login shells- that is far beyond the scope here. Just wanting to clean up the language to make it clear this isn't the only way to set this.